css/tideline-colors.less

```diff
@@ -42,6 +42,8 @@
 @fill-light: #F2F4F6;
 @fill-lightest: #F8F9FA;
 
+@fill-smbg-hover: #A6A6A6;
```

Comment:
I'm not fond of this variable name. As far as I can tell, this color is only used for the color of the `min` and `max` text in the overlays...??

The `@fill-` prefix in the tideline colors only refers to the background rectangles. I may do a bit of CSS re-org to eliminate the need for this extra variable.